src/Tizen.NUI/src/internal/Common/PropertyHelper.cs

```diff
@@ -109,15 +110,14 @@ private static SearchResult GenerateVisualPropertySearchResult(View view, Visual
             if (property.propertyIndex == Property.InvalidIndex)
             {
                 property.Dispose();
-                return null;
+                return data.RelatedData == null ? null : GenerateVisualPropertySearchResult(view, data.RelatedData);
             }
 
             SearchResult result = new SearchResult(property, data.ObjectConverter, data.PropertyValueConverter);
 
-            while (data.RelatedData != null)
+            if (data.RelatedData != null)
```

Comment:
this looks to change the behavior of the `while() ... data = data.RelatedData` 
is this OK?

Reply:
Yes :) The previous code has a bug. It need to be fixed to remove `while`.

Reply:
OK, I got it~